inventory/rh-sso/group_vars/osp-provisioner.yml

```diff
@@ -34,6 +34,11 @@ osp_security_groups:
 - name: "sso-sec-group"
   description: "My sso Sec Group"
   rules:
+  - protocol: tcp
+    port_range_min: 8443
+    port_range_max: 8443
+    direction: ingress
+    remote_ip_prefix: 0.0.0.0/0
   - protocol: tcp
     port_range_min: 8080
```

Comment:
Do we actually need to open up 8080 anymore if we're going to be defaulting to https?

Reply:
That's correct. We don't need to expose this traffic. I will address that.